api/src/wfl/api/spec.clj

```diff
@@ -30,13 +30,16 @@
 (s/def ::uuid-query (s/keys :opt-un [::uuid]))
 (s/def ::version string?)
 (s/def ::wdl string?)
-(s/def ::workload-request (s/keys :req-un [::cromwell
-                                           ::input
+(s/def ::workload-request (s/keys :opt-un [::input
+                                           ::common_inputs]
+                                  :req-un [::cromwell
                                            ::items
                                            ::output
                                            ::pipeline
                                            ::project]))
 (s/def ::workload-response (s/keys :opt-un [::finished
+                                            ::input
+                                            ::options
```

Comment:
Are `options` the same thing as workflow options here?

Reply:
They don't exist! I'll remove :)